package.json

```diff
@@ -111,7 +111,7 @@
     "core-js": "3.38.1",
     "d3": "5.16.0",
     "deep-freeze": "0.0.1",
-    "franklin-sites": "0.0.247",
+    "franklin-sites": "file:.yalc/franklin-sites",
```

Comment:
I'm guessing this is not meant to go in the final code for this PR

Reply:
Not sure how that got in there!